### DISCUSSION TITLE
more cursor trail customization

### DISCUSSION THREAD

**Discussion by Fliperdos** [ppy/osu#21023] [Ideas] [answered]
recently stumbled across this game called sound space+ and i really liked how you could customize more the cursor trail and i think it would be cool if this was in osu!lazer too
heres a gameplay video for that game:
https://m.youtube.com/watch?v=fhsU4gqB8h0

**Comment by mk56-spn** [comment]
dont see the point in linking a video for that game if its not going to showcase the cursors that youd consider nice to have in lazer, that aside more cursor customisation will be coming at some point yes

**Comment by peppy** [comment, marked as answer]
We plan on having more cursor customisation once skinnable gameplay configuration comes online.
